lib/values/enums.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:podcasty/modules/auth/utils/auth_multiple_validators.dart';
+import 'package:podcasty/values/common_validators.dart';
+import 'package:podcasty/values/string_constants.dart';
+
+enum ValidationType {
+  name,
```

Comment:
Please rename this as it is provided in the sdk.

Reply:
Resolved